README.md

```diff
@@ -82,6 +82,7 @@ Rule                              | Recommended      | Options
 [no-unsafe-spy][]                 | 1                |
 [valid-expect][]                  | 1                |
 [prefer-jasmine-matcher][]        | 1                |
+[prefer-toHaveBeenCalledWith][]   | 0                |
```

Comment:
we should set this to warn, using `toHaveBeenCalled` is a bad decision 99.9% of the time